frontend/PreferencesPanel.js

```diff
@@ -90,9 +90,10 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
               {hasCustomTheme && (<option value={CUSTOM_THEME_NAME}>Custom</option>)}
-              {(!showHiddenThemes || hasCustomTheme) && <option disabled="disabled">---</option>}
+              {(browserName || !showHiddenThemes || hasCustomTheme) &&
```

Comment:
Given that the above conditional changed from `!showHiddenThemes` to `browserName` - I think this check should change to just `(browserName || hasCustomTheme)` ?